Vagrantfile

```diff
@@ -1,10 +1,13 @@
+
 hostname = "atlantis-aquarium-vm"
 domain   = "local"
 
 Vagrant.configure("2") do |config|
+  config.ssh.private_key_path = "./vagrant-id_rsa"
```

Comment:
the vagrant box has a customized key for vagrant user (the key generated in provision.rb) so the default does not work in this case